src/main/js/isd.js

```diff
@@ -59,23 +59,59 @@
 
         };
 
-        /* process regions */
+        /* Filter body contents - Only process what we need within the offset and discard regions not applicable to the content */
+        var body = {};
+        var activeRegions = new Set();
+
+        function filter(offset, element) {
+            function offsetFilter(element) {
+                return !(offset < element.begin || offset >= element.end);    
+            }    
+        
+            if (element.contents) {
```

Comment:
There is an uncaught exception when rendering https://github.com/w3c/imsc-tests/blob/main/imsc1/ttml/structure/Structure002.ttml

`Uncaught TypeError: Cannot read property 'contents' of null`

Reply:
I see, that test omits the `body` element and we don't check for a non-existent `body`. Will fix.

Reply:
Fixed in 8cf817f.